### ISSUE TITLE
Memory leak issue in some part of code

### ISSUE THREAD

**Issue by HimanshuS01** [systers/powerup-android#321]
There is some part of code in following activities that causes the low memory problem:

```
1. AvatarRoomActivity.java
2. DressingRoomActivity.java
3. GameActivity.java
4. ScenarioOverActivity.java
5. SelectFeaturesActivity.java
```

This problem is basically caused by holding onto context classes in static fields.As long as that reference exists, the Activity will be kept in memory, leaking all of its views and hence not collected by GC  in android.

@chhavip @anubhakushwaha @aanchal07 If this is a valid issue should I send PR for the same?

**Comment by anubhakushwaha** [comment]
@HimanshuS01 Yes go ahead.

**Comment by HimanshuS01** [comment]
sure @anubhakushwaha . Thanks :)

**Comment by anubhakushwaha** [comment]
Fixed via #321
